Specification/2._Basic_concepts.md

```diff
@@ -26,30 +26,30 @@ inside the same network.
 
 Message and service data structures are defined using the
 [*data structure description language* (DSDL)](/Specification/3._Data_structure_description_language).
-DSDL description is used to generate the serialization/deserialization code for a given data structure for the target
+The DSDL description is used to generate the serialization/deserialization code for a given data structure in each target
 programming language.
-The DSDL approach allows to determine the data structure size statically, thus helping to optimize the protocol
+The DSDL approach allows compilers to determine the data structure size statically, thus helping to optimize the protocol
 implementations in terms of memory consumption and performance.
 This feature is important for deeply embedded systems, where the memory footprint is critical and dynamic memory
 allocation may not be acceptable.
 
-On top of the standard data types, UAVCAN defines a set of standard high-level functions like node health monitoring,
-network discovery, time synchronization, firmware update, and more. There's a part of the specification dedicated to
+On top of the standard data types, UAVCAN defines a set of standard high-level functions including: node health monitoring,
+network discovery, time synchronization, firmware update, and more. For more information see the part of the specification dedicated to
 the [standard data types and application level functions](/Specification/6._Application_level_functions).
 
 Serialized message and service data structures are exchanged by means of the
 [CAN bus transport layer](/Specification/4._CAN_bus_transport_layer), which implements automatic decomposition of long
-transfers into several CAN frames, allowing to exchange data structures of arbitrary size.
+transfers into several CAN frames, allowing nodes to exchange data structures of arbitrary size.
 
 ## Message broadcasting
 
 Message broadcasting refers to the transmission of a serialized data structure over the CAN bus to other nodes
 that are interested to receive this data structure.
```

Comment:
I'm not sure on this one:
1. You've already introduced the fact that there are publish/subscribe semantics above (to differentiate between the two types). I'm not sure we should say less here.
2. I've just "scanned" chapter 4 and while I may have missed something, I still can't tell what the reception semantics are. 

How we handle this depends on whether how receivers work. Either:
1. there is a registration process. What am I missing?
2. There is no registration process - broadcast is to everyone and nodes choose to act on or ignore messages individually based on their own internal logic.